packages/renderer/src/stitcher.ts

```diff
@@ -38,10 +39,16 @@ export const stitchFramesToVideo = async (options: {
 			`${options.width}x${options.height}`,
 			'-i',
 			`element-%0${numberLength}d.${format}`,
-			'-vcodec',
-			'libx264',
+			options.outputFormat === 'mp4' ? '-vcodec' : '-c:v',
```

Comment:
If I understand correctly, `-vcodec` and `-c:v` is exactly the same :)

Reply:
Oh, good find @yuta0801! Can we remove this ternary statement @Iamshankhadeep? Doesn't matter which one we use

Reply:
Yeah my bad I thought I removed it. But thanks again.